tools/jenkins/run_distribution.sh

```diff
@@ -134,12 +124,11 @@ elif [ "$platform" == "macos" ]; then
         ;;
     esac
 
-    # clean up
-    cd ~/ 
+    # Clean up
     rm -rf $brew_root
 
     # Make sure the system brew installation is still unaffected
-    export PATH=$OLD_PATH
+    export PATH=$ORIGINAL_PATH
```

Comment:
I wanted to run `brew list -l` on the original brew installation at the end of this script just to make sure the system homebrew is still fine. So I need to reset my `$PATH` so that I find the original `brew` command.